src/strings/csv/es.csv

```diff
@@ -386,6 +386,9 @@ GRAMINOID,Gramínea
 GRAMS,Gramos
 GREATER_THAN_FIFTY_PERCENT,> 50%
 GROWTH_FORM,Forma de crecimiento
+HIGHEST,Más alta
+HIGHEST_AND_LOWEST_MORTALITY_RATE_CARD_TITLE,¿Qué zonas tienen la tasa de mortalidad más alta y más baja?
+HIGHEST_AND_LOWEST_MORTALITY_RATE_SPECIES_CARD_TITLE,¿Qué especie tienen la tasa de mortalidad más alta y cuál la más baja?
```

Comment:
I would say 'Que especie tiene.." tiene without the 'n'

Reply:
It's incorrect how it is. It could also be "¿Qué especies tienen ... " adding an "s" to "especie" but it gives the idea that more than one species will have the highest morality rate, and we are showing only one.

Reply:
Ok, thanks. We need help updating this in the phrase portal perhaps. @alexTerraformation @sgrimm